pkg/controllers/user/logging/deployer/testerdeployer.go

```diff
@@ -48,7 +50,7 @@ func (d *TesterDeployer) Deploy(level, clusterName, projectID string, loggingTar
 	}
 
 	systemProjectID := ref.Ref(systemProject)
-	creator, err := d.systemAccountManager.GetProjectSystemUser(systemProject.Name)
+	creator, err := d.systemAccountManager.GetSystemUser(systemProject.Spec.ClusterName)
```

Comment:
can you explain why you need to switch to the cluster-wide user rahter than project user?

Reply:
Hi, this app will deployed to system project, and the chart has clusterrole rbac, project user couldn't create clusterroles rbac.
https://github.com/rancher/system-charts/blob/dev/charts/rancher-logging/0.1.1/charts/fluentd-tester/templates/clusterrole.yaml

Reply:
That is a good explanatio. Can you add that to the commit message? You don't need to reference the chart via a github link, but just add something like:
```
This app will deployed to system project, and the chart has clusterrole rbac, project user couldn't create clusterroles rbac.
```

Reply:
Updated. Thanks